__tests__/app.test.js

```diff
@@ -65,23 +65,83 @@ describe("app", () => {
                 expect(typeof res.body.topics).toBe("object")
                 expect(res.body.topics).toHaveLength(3)
             })
+        });
+    })
+
+    describe("GET /api/articles/:article_id/comments", () => {
+        test("should return an array of objects with expected properties", () => {
+            return request(app).get('/api/articles/1/comments').expect(200).then((res) => {
+                const result = res.body.comments
+                result.forEach((element) => {
+                    if (result.length != 0) {
+                        expect(element).toMatchObject({
+                            comment_id: expect.any(Number),
+                            votes: expect.any(Number),
+                            created_at: expect.any(String),
+                            author: expect.any(String),
+                            body: expect.any(String),
+                            article_id: expect.any(Number)
+                        })
+                    }
+                });
+            })
+        });
 
+        test("should return an array of objects in order", () => {
+            return request(app).get('/api/articles/1/comments').expect(200).then((res) => {
+                const result = res.body.comments
+                expect(result).toBeSortedBy('created_at', {
+                    descending: true,
+                    coerce: true,
```

Comment:
Since they're the same data type, you shouldn't need to coerce. just descending should be perfectly fine